build/xtask/src/clippy.rs

```diff
@@ -16,7 +16,7 @@ pub fn run(
 ) -> Result<()> {
     let toml = Config::from_file(&cfg)?;
 
-    let mut src_dir = cfg.to_path_buf();
+    let mut src_dir = cfg;
```

Comment:
is there a reason to prefer this over making `cfg` `mut` above?

Reply:
I decided on
```
    let src_dir = cfg
        .parent()
        .ok_or_else(|| anyhow!("Could not get src dir"))?;
```
which has nothing mutable!